django_pandas/io.py

```diff
@@ -21,12 +21,7 @@ def to_fields(qs, fieldnames):
                             model = field.model
                             break
             else:
-                if (hasattr(field, "one_to_many") and field.one_to_many) or \
-                   (hasattr(field, "one_to_one") and field.one_to_one):
-                    model = field.related_model
-                elif field.get_internal_type() in (
-                        'ForeignKey', 'OneToOneField', 'ManyToManyField'):
-                    model = field.rel.to
+                model = get_related_model(field)
```

Comment:
@chrisdev Do we need to check if get_related_model returns `None` here?

Reply:
Not sure We can always do a patch release if necessary